cmd/storj-sim/network.go

```diff
@@ -171,6 +171,8 @@ func newNetwork(flags *Flags) (*Processes, error) {
 
 				"--server.extensions.revocation=false",
 				"--server.use-peer-ca-whitelist=false",
+
+				// TODO(yar): add mail configuration
```

Comment:
Maybe for testplanet and satellite we could make it auto-approve with some flag i.e. avoid sending an email out? Any other alternative (like setting up custom server and client etc.) seems very messy.

Reply:
It makes sense if we turn off account activation as well, otherwise satellite can be only used with test account we created for integration testing. Other option is, as we have two auth type supported, to use plain auth as default configuration to omit all oaaut2 info here.